#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_perl_bindings_prefix.dpatch by laga <>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Force PERL bindings to be installed into $PREFIX

@DPATCH@

Index: bindings/perl/perl.pro
===================================================================
--- a/bindings/perl/perl.pro      (revision 13967)
+++ b/bindings/perl/perl.pro      (working copy)
@@ -4,12 +4,12 @@

 mythperlbindings.target = Makefile.perl
 mythperlbindings.depends = Makefile.PL
-contains(PREFIX, ^/usr(/local)?/?$) {
-    mythperlbindings.commands = perl Makefile.PL MAKEFILE=Makefile.perl
-}
-else {
+#contains(PREFIX, ^/usr(/local)?/?$) {
+#    mythperlbindings.commands = perl Makefile.PL MAKEFILE=Makefile.perl
+#}
+#else {
     mythperlbindings.commands = perl Makefile.PL MAKEFILE=Makefile.perl PREFIX=$${PREFIX}
-}
+#}

 mythperbindingsbuild.target = perl_build
 mythperbindingsbuild.depends = Makefile.perl


